packages/debugger/package.json

```diff
@@ -25,6 +26,9 @@
     "@radix-ui/react-slot": "^1.0.2",
     "@radix-ui/react-switch": "^1.0.3",
     "@radix-ui/react-tabs": "^1.0.4",
+    "@rainbow-me/rainbowkit": "^2.0.2",
+    "@reservoir0x/reservoir-sdk": "^2.0.11",
+    "@tanstack/react-query": "^5.22.2",
```

Comment:
is this dependency necessary?

Reply:
it's a wrapper for the api call, but it contains an up to date list of chains and their corresponding API urls - the footprint is minimal https://bundlephobia.com/package/@reservoir0x/reservoir-sdk@2.0.11

Reply:
I meant `@tanstack/react-query`. I don't see it used here.

Reply:
it's a peer dependency of `@rainbow-me/rainbowkit`